pages/task/read/views/project.jsx

```diff
@@ -144,7 +146,7 @@ const Project = ({ task, project, establishment, children, schema, formFields })
       }
 
       {
-        completeAndCorrect(task, project) &&
+        !isAmendment && completeAndCorrect(task, isAmendment) &&
```

Comment:
Do we use `completeAndCorrect` anywhere _except_ immediately after checking it's not an amendment?

If not then the second argument might be redundant.

Reply:
Yeah I was debating this, as it's currently only called for new applications. 

But if we called `allDeclarationsConfirmed` somewhere else in future without realising we need to deal with `isAmendment` then we'll end up with a bug, so I thought it best to make it obvious while it's fresh in my mind.

Reply:
That actually seems pretty reasonable. 

👍